reV/bespoke/cli_bespoke.py

```diff
@@ -130,9 +130,11 @@ def from_config(ctx, config_file, points_range, verbose):
 
     is_multi_node = (config.execution_control.option in ('eagle', 'slurm')
                      and config.points_range is None)
-    workers = config.execution_control.nodes if is_multi_node else 1
+    workers = config.execution_control.max_workers if is_multi_node else 1
```

Comment:
I actually think this was correct before. Do you have a reason for changing this besides the (shitty) naming convention? I can explain the parallelization scheme if you want.

Reply:
Yea, I have the following exec control in my config file:
```"execution_control": {
	...
	"max_workers": 36,
	"nodes": 1,
	"memory": 79
	...
}```
So I was asking for 36 threads on 1 cpu, and with the code as it was, it was only opening 1 thread on 1 cpu. Are you suggesting that I should instead request 1 thread on 36 nodes? Perhaps an explanation of the parallelization scheme would help.

Reply:
Ah, taking a closer look at the run logs, I see that even with this change, I checked out 36 nodes. SO it seems I really should be using 
```
"execution_control": {
	...
	"nodes": 36,
	"memory": 79
	...
}
```
Is there any point where the `max_workers` input is actually useful?

Reply:
yeah so max_workers defaults to None which uses all available CPU's on a node. If you are running out of memory you'll reduce max_workers or if you're debugging you'll set max_workers=1 for serial and more coherent logging. 

I think max_workers is fine in the rest of the CLI. You only need the nodes argument in that one spot and then the CLI has to pass max_workers down all the way to the Bespoke class where the process pool starts on a single node.